pipelines/LSSTCam-imSim/ApPipe.yaml

```diff
@@ -14,3 +14,4 @@ tasks:
     config:
       # Alert distribution only runnable in PP environment.
       alertPackager.doProduceAlerts: True
+      config.astrometry.referenceSelector.doUnresolved: False
```

Comment:
Is the leading `config.` necessary?

Reply:
Agree that the leading `config.` should not be there, but more importantly this is a config for `calibrateImage` not `diaPipe`. So to set this you would need to exclude `calibrateImage` from `ApPipe.yaml` above and redefine it here, along with all of it's custom connections and configs.